README.md

```diff
@@ -77,3 +78,65 @@ In order to expose those ports outside the local environment, you can edit lines
       - "${SCNODE_REST_PORT}:${SCNODE_REST_PORT}"
    ```
+
+## Running a Forger Node 
+Should you want to run a forger node, please select "forger" when you run the init.sh script for the first time. Keep in mind it will take time for the Mainchain node and the sidechain node to fully sync the whole chains.
+
+Step by step guide : 
+
+1 - On init.sh script select "forger" when prompted. 
+2 - Once the Evm node and the Mainchain node are up&running, run the generate_forger_keys script 
+    ```shell
+    ./scripts/generate_forger_keys.sh
+    ```
+    The script will generate for you all the necessary keys on the local node. The script output will be 
+    ```shell
+      Forger Keys (Generated Vrf public key)
+      blockSignPublicKey (Generated blockSignPublicKey)
+      ethereum compatible address key pair (Generated Ethereum address)
+    ```
+
+3 - At this point, in order to stake on EON via smart contract, you will create a Forging Stake using Web3js; 
+    Here is an example (to be run in the Remix IDE) of creating the forging stake with a smart contract interaction:
+    ```shell
+(async () => {
+    try {
+
+        console.log('Running testWeb3 - Delegate or create a new Forging Stake script...')
+
+        const smartContractAddress = "0x0000000000000000000022222222222222222222";
+        const amount = 0.001; // amount of ZEN of the new stake
+        const accounts = await web3.eth.getAccounts();
+        console.log('Account ' + accounts[0]);
+
+        const abi = require("browser/contracts/artifacts/abi_fs.json"); // Change this for different path
+
+        const contract = new web3.eth.Contract(abi, smartContractAddress, {from: accounts[0]});
+
+        // Testing method:  delegate
+        const ownerAddress = accounts[0];
+        // pick one of the existing forgers, or create a new forger
+        const blockSignPublicKey = "0x" + YOUR_BLOCK_SIGN; // the address that will sign the block when forged; you have to populate this with the value "blockSignPublicKey" you received in the previous step
+        const forgerVrfPublicKey = "0x" + YOUR_VRF_KEY; // this is the key created in the previous step (Forger Keys)
+        const first32BytesForgerVrfPublicKey = forgerVrfPublicKey.substring(0, 66);
+        const lastByteForgerVrfPublicKey = "0x" + forgerVrfPublicKey.substring(66, 68);
+
+        methodName = 'delegate';
+        console.log('Response for '+ methodName);
+
+        const response = await contract.methods.delegate(blockSignPublicKey,first32BytesForgerVrfPublicKey,lastByteForgerVrfPublicKey,ownerAddress).send({value: amount *10**18}).then(console.log);
+
+    } catch (e) {
+        console.log("Error:" + e.message);
+    }
+  })()
+        ```
+
+⚠️ Remember to replace YOUR_BLOCK_SIGN and YOUR_VRF_KEY with the actual keys, and to specify the path of the native contract ABI. The contract ABI is not provided here, but it is available with the complete Remix workspace.
```

Comment:
"with the actual keys you have generated by running  'generate_forger_keys.sh' script. You will also need to provide a path to the abi contract file location which is not provided here, but is available with the complete Remix workspace."